.github/workflows/release-winget.yml

```diff
@@ -0,0 +1,30 @@
+name: Submit Winget Package to Windows Package Manager Community Repository
+on:
+  release:
+    types: [released]
+  workflow_dispatch:
+    inputs:
+      tag_name:
+        description: "Specific tag name"
+        required: false
+        type: string
+env:
+  CI: true
+jobs:
+  winget-publish:
+    name: Publish winget package
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set winget version env
+        env:
+          TAG_NAME: ${{ inputs.tag_name || github.event.release.tag_name }}
+        run: echo "WINGET_TAG_NAME=$(echo ${TAG_NAME#v})" >> $GITHUB_ENV
+      - name: Submit package to Windows Package Manager Community Repository
+        uses: vedantmgoyal2009/winget-releaser@v2
+        with:
+          identifier: Triplex.Triplex
+          installers-regex: '^Triplex-Setup-[\d\.]+\.exe$'
+          version: ${{ env.WINGET_TAG_NAME }}
```

Comment:
Do we need to have these env vars set or they get set by the action?

Reply:
There are two ways to triger the action: 
 - one is automatic on every release, github populates that env variable from `github.event.release.tag_name`
 - or on manual trigger, this has to be input by the user doing the manual winget-release, `inputs.tag_name`